CHANGELOG.adoc

```diff
@@ -0,0 +1,34 @@
+= Changelog
```

Comment:
I had to add a changeset as the CI doesn't pick up base branch changes and won't execute otherwise